Makefile.am

```diff
@@ -21,6 +21,7 @@ EXTRA_DIST		= CONTRIBUTING.md		\
 			  m4/gnulib-tool.m4		\
 			  m4/PKG_CHECK_VAR.m4		\
 			  m4/REQUIRE_HEADER.m4		\
+			  pakemaker.sysusers		\
```

Comment:
In slack, you said it was more difficult to put this file in rpm/ than expected.  What's the problem you're seeing?

Reply:
when calling rpmbuild it has to be where the tar-file is.

Reply:
That said it doesn't show up there automatically anymore and we have to dynamically create (copy in that case) it. But having to dynamically create it anyway I'd then rather go for creating it from pacemaker.spec.in keeping that as the only source for IDs and names.